src/ray/raylet/scheduling_resources.cc

```diff
@@ -17,14 +17,14 @@ ResourceSet::ResourceSet(const std::vector<std::string> &resource_labels,
                          const std::vector<double> resource_capacity) {
   RAY_CHECK(resource_labels.size() == resource_capacity.size());
   for (uint i = 0; i < resource_labels.size(); i++) {
-    RAY_CHECK(this->AddResource(resource_labels[i], resource_capacity[i]));
+    RAY_CHECK(AddResource(resource_labels[i], resource_capacity[i]));
   }
 }
 
 ResourceSet::~ResourceSet() {}
 
 bool ResourceSet::operator==(const ResourceSet &rhs) const {
-  return (this->IsSubset(rhs) && rhs.IsSubset(*this));
+  return (IsSubset(rhs) && rhs.IsSubset(*this));
```

Comment:
actually, in the case where we overload binary operators, it's useful to keep `this->` for code readability. You get "this" is a subset of the right hand side. Removing it makes it a little less clear what `rhs` is supposed to be a subset of (because `this->` is implicit).